pkg/cdn/metadata/rdbms/models/cdn_metadata_entry.go

```diff
@@ -0,0 +1,50 @@
+package models
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/gobuffalo/pop"
+	"github.com/gobuffalo/uuid"
+	"github.com/gobuffalo/validate"
+	"github.com/gobuffalo/validate/validators"
+)
+
+// CdnMetadataEntry stores the module name and cdn URL.
+type CdnMetadataEntry struct {
+	ID          uuid.UUID `json:"id" db:"id"`
+	CreatedAt   time.Time `json:"created_at" db:"created_at"`
+	UpdatedAt   time.Time `json:"updated_at" db:"updated_at"`
+	Module      string    `json:"module" db:"module"`
+	RedirectURL string    `json:"redirect_url" db:"redirect_url"`
+}
+
+// String is not required by pop and may be deleted
+func (e CdnMetadataEntry) String() string {
+	je, _ := json.Marshal(e)
```

Comment:
m: maybe just log the error? it does not feel right to ignore

Reply:
It's the model code auto-generated  by pop. Should I log this in the storage pkg as well?

Reply:
guess not, i did not realized that it is autogenerated